server/task-cassandra/src/main/scala/org/apache/james/task/eventsourcing/cassandra/TaskEventsDTO.scala

```diff
@@ -0,0 +1,115 @@
+/** **************************************************************
+  * Licensed to the Apache Software Foundation (ASF) under one   *
+  * or more contributor license agreements.  See the NOTICE file *
+  * distributed with this work for additional information        *
+  * regarding copyright ownership.  The ASF licenses this file   *
+  * to you under the Apache License, Version 2.0 (the            *
+  * "License"); you may not use this file except in compliance   *
+  * with the License.  You may obtain a copy of the License at   *
+  *                                                              *
+  * http://www.apache.org/licenses/LICENSE-2.0                   *
+  *                                                              *
+  * Unless required by applicable law or agreed to in writing,   *
+  * software distributed under the License is distributed on an  *
+  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+  * KIND, either express or implied.  See the License for the    *
+  * specific language governing permissions and limitations      *
+  * under the License.                                           *
+  * ***************************************************************/
+
+package org.apache.james.task.eventsourcing.cassandra
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import org.apache.james.eventsourcing.EventId
+import org.apache.james.eventsourcing.eventstore.cassandra.dto.EventDTO
+import org.apache.james.server.task.json.JsonTaskSerializer
+import org.apache.james.task.{Task, TaskId}
+import org.apache.james.task.eventsourcing._
+
+sealed abstract class TaskEventDTO(val getType: String, val getAggregate: String, val getEvent: Int) extends EventDTO {
```

Comment:
I'm no Scala expert but is it normal the name of the file is slightly different with the name of the class?
`TaskEventsDTO` vs `TaskEventDTO` ? (`s` diff at Event)